.github/workflows/build.yml

```
@@ -267,6 +267,9 @@ jobs:
         shell: pwsh
         run: ./GenerateAllSolution.ps1
 
+      - name: Install ninja
+        run: apt-get install ninja-build
```

```suggestion
      - name: Install ninja for WASM native dependencies
        run: sudo apt-get install ninja-build
```

Add sudo command for install (and improve description). @Arlodotexe will this effect codespaces?

---

`./.github/workflows/build.yml` is only used by GitHub Actions. This should be safe to do in Actions, according to the [docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).

> The Linux and macOS virtual machines both run using passwordless sudo. When you need to execute commands or install tools that require more privileges than the current user, you can use sudo without needing to provide a password.

To install this in Codespaces, modify the Docker config at `./.devcontainer/Dockerfile`.